spec/requests/api/v1/dish_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::DishesController, type: :request do
+  describe 'GET /v1/dishes' do
+    let!(:dish) {FactoryBot.create(:dish, name: 'dumplings', price: 89)}
```

Comment:
Please add FactoryBot matchers to `rails_helper` to be able to shorten this syntax

Reply:
I would also create a set of dishes here. Like 4 or 5 and test that all of them are included in the response.